packages/core/src/common/preferences/preference-scope.ts

```diff
@@ -57,7 +57,7 @@ export namespace PreferenceScope {
             case 'application':
                 return PreferenceScope.User;
             case 'window':
-                return PreferenceScope.Workspace;
+                return PreferenceScope.Folder;
```

Comment:
hello, in which case do we have preferencescope.workpace ?

Reply:
it is inclusive, i.e folder allows workspace and global

Reply:
does it mean, users wouldn't be able to define prefs only available at the "workspace" & "user" level?

Reply:
actually the whole scope is valid check is bogus right now, it should be aligned with: 
https://github.com/Microsoft/vscode/blob/750680d2f7ae563529501f8ec419d83ccf8ec325/src/vs/workbench/services/configuration/common/configuration.ts#L23-L26

It is not inclusive,  folder preferences allow ONLY resource, workspace preference allow ONLY  window and resource level. I will rework it as well.

Reply:
@elaihau @benoitf Would be fine to fix it with a follow-up PR? otherwise it would be postponed more and it does not seem to relate to referenced issues.

Reply:
yes

Reply:
yes it could be resolved in a separate PR. since the next release would be either on Apr 30 or May 1, do we want to hold this one until the release is made?

Reply:
please ask on Spectrum, if someone has concrete objections, yes

from my side, most important that it should be tested not only by me, but double checked by someone else